pages/search.js

```diff
@@ -244,6 +232,12 @@ export default function Search({ data, queryString }) {
         <ChipDataView chipData={chipData}></ChipDataView>
         <AccessibilityDataView data={dnaseData}></AccessibilityDataView>
         <QTLDataView data={QTLData}></QTLDataView>
+        <Motifs
+          motifsList={motifDocList}
+          sequence={data.sequence}
+          coordinates={coordinates}
+          assembly={data.assembly}
+        ></Motifs>
```

Comment:
You can just use a self-closing component, like `<Motifs … />`. I enabled a rule enforcing this in igvf-ui. It’s a preference thing.